templates/digital-asset-template/frontend/src/App.tsx

```diff
@@ -1,20 +1,183 @@
 import { WalletSelector } from "@aptos-labs/wallet-adapter-ant-design";
 import "@aptos-labs/wallet-adapter-ant-design/dist/index.css";
-import logo from "./assets/aptos.png";
+import { Button } from "./components/ui/button";
+import { Input } from "./components/ui/input";
+import { Progress } from "./components/ui/progress";
+import { config } from "./config";
+
+const IMG = (w: number = 300, h: number = 300) =>
+  `https://picsum.photos/seed/picsum/${w}/${h}`;
+
+const COLLECTION_ADDRESS = "0x8973..6855";
+const COLLECTION_NAME = "Forest Friends";
+const COLLECTION_DESCRIPTION = `Step into the enchanting world of "Forest Friends," a captivating NFT collection that celebrates the beauty of nature through charmingly crafted animal portraits. Each NFT in this collection showcases a unique forest inhabitant, rendered with exquisite detail and vibrant colors that breathe life into the digital canvas. From wise owls exuding a sense of serenity to playful foxes radiating boundless energy, every portrait captures the diverse range of emotions and personalities found in the heart of the wilderness.`;
 
 function App() {
   return (
-    <>
-      <div className="navbar">
-        <div className="navbar-text">Create Aptos Dapp</div>
+    <main className="flex flex-col gap-6">
+      {/* Nav Bar */}
+      <nav className="navbar-container px-6 flex justify-between">
+        <p className="navbar-text">Forest Friends</p>
         <div>
           <WalletSelector />
         </div>
-      </div>
-      <div className="center-container">
-        <img className="center-image" src={logo} alt="aptos" />
-      </div>
-    </>
+      </nav>
+
+      {/* Hero */}
+      <section className="hero-container flex gap-6 px-6">
```

Comment:
I would break each of these sections out into separate components. A few reasons why:

- You can assign a name to each section symbolically instead of placing a comment above it.
- Your `App` component will read more like a table of contents and users can quickly jump to any section instead of having to scroll to find it. Additionally, this will make it easier for users to comment out or delete sections.
- The secret to keeping Tailwind manageable is small components. All of the utility classes can easily create a wall of unreadable markup. But if you keep your components small, you can essentially tuck that `className` complexity away into tiny pockets.

For now, each section component can probably just live in this file. Depending on how they grow, they each may become worthy of their own file down the road.